tests/group_vars/all.yml

```diff
@@ -28,7 +28,11 @@ dot_env:
     - { key: 'pusher_key', value: 'test' }
     - { key: 'pusher_secret', value: 'test' }
 
+laravel_access_control_allow_credentials: false
+laravel_access_control_allow_headers: 'DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type'
+laravel_access_control_allow_methods: 'GET,POST,OPTIONS'
```

Comment:
? Arrange the headers alphabetically so it will be easier to look for something in the future.